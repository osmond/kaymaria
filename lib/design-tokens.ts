export const lightTokens = {
  primary: "#508C7E",
  secondary: "#D3EDE6",
  background: "#F9F9F9",
  foreground: "#111827",
  muted: "#9CA3AF",
  border: "#E5E7EB",
  success: "#16A34A",
  warning: "#F59E0B",
  destructive: "#DC2626",
};

export const darkTokens = {
  primary: "#508C7E",
  secondary: "#D3EDE6",
  background: "#0B0F0E",
  foreground: "#E5E7EB",
  muted: "#9CA3AF",
  border: "#1F2A28",
  success: "#16A34A",
  warning: "#F59E0B",
  destructive: "#DC2626",
};

export const coreColors = lightTokens;
