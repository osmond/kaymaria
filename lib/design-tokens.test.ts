/**
 * @jest-environment node
 */
import { lightTokens, darkTokens } from './design-tokens';

describe('design tokens', () => {
  it('matches light token snapshot', () => {
    expect(lightTokens).toMatchSnapshot();
  });
  it('matches dark token snapshot', () => {
    expect(darkTokens).toMatchSnapshot();
  });
});
