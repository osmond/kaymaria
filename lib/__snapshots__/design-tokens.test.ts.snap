// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`design tokens matches dark token snapshot 1`] = `
{
  "background": "#0B0F0E",
  "border": "#1F2A28",
  "destructive": "#DC2626",
  "foreground": "#E5E7EB",
  "muted": "#9CA3AF",
  "primary": "#508C7E",
  "secondary": "#D3EDE6",
  "success": "#16A34A",
  "warning": "#F59E0B",
}
`;

exports[`design tokens matches light token snapshot 1`] = `
{
  "background": "#F9F9F9",
  "border": "#E5E7EB",
  "destructive": "#DC2626",
  "foreground": "#111827",
  "muted": "#9CA3AF",
  "primary": "#508C7E",
  "secondary": "#D3EDE6",
  "success": "#16A34A",
  "warning": "#F59E0B",
}
`;
